src/components/chapters/ChapterCard.tsx

```diff
@@ -0,0 +1,126 @@
+import {
+  Box,
+  Heading,
+  Text,
+  Link,
+  HStack,
+  Tooltip,
+  VStack,
+  Tag,
+} from "@chakra-ui/react";
+import { AiFillFacebook } from "react-icons/ai";
+import { BsCircleFill } from "react-icons/bs";
+import { FiInstagram } from "react-icons/fi";
+import { IoIosCheckmarkCircle, IoIosGlobe } from "react-icons/io";
+import { Chapter } from "src/utils/types";
+
+function ChapterCard(props: {
+  chapter?: Chapter;
+  isSelected: boolean;
+  onClick: (id: string) => void;
+}) {
+  const { chapter, isSelected, onClick: _onClick } = props;
+  const open = true; // TODO can't calculate from given information
```

Comment:
@mjbernet3 See here :-)

Reply:
Ah yeah.  We need an efficient way to track this.  I'll address this, not going to block the merge for this since the focus of this task is frontend.